src/Behavior.js

```diff
@@ -116,7 +116,7 @@ function toggle_reaction_hover(on_off) {
   if (on_off) {
     this.reaction_mouseover = (d) => {
       if (!this.dragging) {
-        this.map.callback_manager.run('show_tooltip', null, 'reaction_label', d);
+        //this.map.callback_manager.run('show_tooltip', null, 'reaction_label', d);
```

Comment:
Can you please explain why this change is necessary? And would it not be better to remove the whole function, since it now doesn't do anything?

Reply:
Done